<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenAuraSDKQtDialogUi</class>
 <widget class="QMainWindow" name="OpenAuraSDKQtDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenAuraSDK</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButtonSetAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSetColors">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Colors</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED0R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED0G">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED0B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED1R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED1G">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED1B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED2R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED2G">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED2B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED3R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED3G">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED3B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED4R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED4G">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLED4B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGreen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBlue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboAuraDevices">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAuraDevice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aura Device</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GroupBoxDirect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QRadioButton" name="RadioDirect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Direct</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioEffect">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Effect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxEffect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>Effect</string>
    </property>
    <widget class="QRadioButton" name="RadioOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioStatic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Static</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioBreathing">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Breathing</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioFlashing">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Flashing</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioSpectrumCycle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Spectrum Cycle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioRainbow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rainbow Wave</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioBreathingSpectrum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Breathing Spectrum</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadioChaseFade">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Chase Fade</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
